backend/src/main/kotlin/dres/api/cli/CompetitionRunCommand.kt

```diff
@@ -54,6 +54,27 @@ class CompetitionRunCommand(internal val runs: DAO<CompetitionRun>) : NoOpCliktC
         }
     }
 
+    /**
+     * Deletes a selected competition run for the current DRES instance.
+     */
+    inner class DeleteRunCommand: CliktCommand(name = "delete", help = "Deletes an existing competition run.") {
+        private val id: Long by option("-r", "--run").long().required()
+        override fun run() {
+            val count = RunExecutor.managers().count { it.runId == id }
+            if (count > 0) {
```

Comment:
A more direct approach would be
```kotlin
if (RunExecutor.managers().any{ it.runId == id }) {
```

Reply:
Ahh thanks. I was looking for something along those lines but .exists() did not work :)